plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/AggregatePluginPrerequisitesNotMetException.java

```diff
@@ -0,0 +1,14 @@
+package io.jenkins.tools.pluginmanager.impl;
+
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+public class AggregatePluginPrerequisitesNotMetException extends RuntimeException {
+
+    public AggregatePluginPrerequisitesNotMetException(Collection<Exception> exceptions) {
+        super("Multiple plugin prerequisites not met:\n" + exceptions.stream().map(Exception::getMessage).collect(Collectors.joining(",\n")));
```

Comment:
could this get some test coverage please? and also manual steps to reproduce / test it?

Reply:
Done in https://github.com/jenkinsci/plugin-installation-manager-tool/pull/298/commits/c3b5e06eb24244407dd38aa8f60125f241faa9d4